uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -92,7 +97,12 @@ define([
         defaultCustomData: function () {
           return {
             comments: [],
-            request_type: 'leave'
+            request_type: 'leave',
+            uploader: FileUpload.uploader({
+              entityTable: 'civicrm_hrleaveandabsences_leave_request',
+              crmAttachmentToken: sharedSettings.attachmentToken
+            }),
+            files: []
```

Comment:
please remember to put the one-liner properties first and then the multi-lines ones later
```js
{
  comments: [],
  files: [],
  request_type: 'leave',
  uploader: 
}